Packs/CommonTypes/IncidentFields/incidentfield-Process_Name.json

```diff
@@ -0,0 +1,27 @@
+{
+    "id": "incident_processname",
+    "version": -1,
+    "modified": "2021-05-30T15:53:44.066994+03:00",
+    "name": "Process Name",
+    "ownerOnly": false,
+    "cliName": "processname",
+    "type": "shortText",
+    "closeForm": false,
+    "editForm": true,
+    "required": false,
+    "neverSetAsRequired": false,
+    "isReadOnly": false,
+    "useAsKpi": false,
+    "locked": false,
+    "system": false,
+    "content": true,
+    "group": 0,
+    "hidden": false,
+    "associatedToAll": true,
```

Comment:
is this by design? just making sure.

Reply:
@Arsenikr , we said put this field in common did not say what should be the association. Should this be only for CB?

Reply:
Process name should be a generic field.